tensorboard/webapp/routes/index.ts

```diff
@@ -27,5 +28,12 @@ export function routesFactory(): RouteDef[] {
       defaultRoute: true,
       deepLinkProvider: new DashboardDeepLinkProvider(),
     },
+    {
+      routeKind: RouteKind.UNKNOWN,
+      path: '/flags/',
+      ngComponent: FeatureFlagPageContainer as Type<Component>,
+      defaultRoute: false,
+      deepLinkProvider: new DashboardDeepLinkProvider(),
```

Comment:
just curious about what happens if we remove deepLinkProvider here. (I did that and it seems to work the same way as before).

Reply:
This allows for parameters. For instance it would allow us to type in /flags/?enableLinkTime. However, since we don't have the routing going we can only navigate here from a button so this does not really make a difference. I would like to leave it in the hopes that one day we do get the routing working.

Reply:
Even if you do find a need for deep links in feature flags, I don't think DashboardDeepLinkProvider will be the one you use (it's specifically for deep links for the dashboard view).

Reply:
Ok. Removed.